sphinx/themes/basic/layout.html

```diff
@@ -105,8 +105,8 @@
 {%- macro css() %}
     <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
-    {%- for cssfile in css_files %}
-    <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
+    {%- for css in css_files %}
+    <link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
```

Comment:
This is a incompatible change.
I think it would be better if `css` object (current implementation is using dict) has `__str__` and return a value of `filename` attribute to keep compatibility.
@tk0miya How do you think?